packages/grafana-ui/src/components/Divider/Divider.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Preview, ArgTypes } from '@storybook/blocks';
```

Comment:
We dont use `Story` and `Preview` right?

Reply:
Nope, I omitted to remove those before, now I removed them.